### PR TITLE
PN-466: Visited UCSC Genome Browser links are displayed in white fonts

### PR DESCRIPTION
**Pull request by veronikaslc**
Fixed

### CODE REVIEW

File: ui/src/main/resources/PhenoTips/SimilarCases.xml [hunk 1]
```diff
@@ -1659,6 +1659,9 @@
     color: $theme.notificationErrorColor;
     font-size: .87em;
 }
+.match-details .variants-data a:visited {
+    color: #000000 !important;
```
Comment (BrianLiSK) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: Isn't this black? Are we sure we don't want dark grey for visited ones?

Comment (marta-) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: The UCSC links are implemented as "secondary action" buttons. You should use the color defined in the color theme.
```suggestion
    color: $theme.buttonSecondaryTextColor !important;
```

File: ui/src/main/resources/PhenoTips/SimilarCases.xml [hunk 2]
```diff
@@ -1659,6 +1659,9 @@
     color: $theme.notificationErrorColor;
     font-size: .87em;
 }
+.match-details .variants-data a:visited {
```
Comment (marta-) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: This is not the right place to fix the issue. It is actually caused by a selector that is too broad in some unrelated code: https://github.com/phenotips/phenotips/blob/master/components/gene-panels/ui/src/main/resources/PhenoTips/PanelsDisplay.xml#L552 . That selector can affect much more than just the UCSC links for variants. The problem should be fixed there directly.

**Comment by marta-** [comment]
Closing this PR since the root cause is in [PhenoTips code ](https://github.com/phenotips/phenotips/blob/master/components/gene-panels/ui/src/main/resources/PhenoTips/PanelsDisplay.xml#L552) and that's where it should be fixed.
